e2e-tests/cypress/tests/integration/channels/messaging/tooltips_on_top_nav_channel_icons_posts_spec.js

```diff
@@ -75,9 +75,9 @@ describe('Messaging', () => {
         cy.uiGetToolTip(testChannel.display_name);
         cy.get('@longChannelAtSidebar').trigger('mouseout');
 
-        // * Check that the Demo plugin tooltip is present
-        cy.get('@channelHeader').find('.fa-plug').should('be.visible').trigger('mouseover');
-        cy.uiGetToolTip('Demo Plugin');
+        // // * Check that the Demo plugin tooltip is present
+        // cy.get('@channelHeader').find('.fa-plug').should('be.visible').trigger('mouseover');
```

Comment:
the demoplugin icon is not showing up/

Reply:
Was the demo plugin installed correctly? Could it be test setup issue or an actual product bug?

Reply:
resolved. It needed some more config settings to be enabled.